appserver/extras/payara-micro/payara-micro-distribution/pom.xml

```diff
@@ -81,26 +79,39 @@
             </executions>
         </plugin>
         <plugin>
-        <groupId>org.jvnet.maven-antrun-extended-plugin</groupId>
-        <artifactId>maven-antrun-extended-plugin</artifactId>
-        <executions>
-            <execution>
-                <phase>package</phase>
-                <goals>
-                    <goal>run</goal>
-                </goals>
-                <configuration>
-                    <tasks>
-                        <ant dir="." antfile="build.xml" target="create.distribution">
-                            <property name="bundlename" value="fish.payara.micro" />
-                            <property name="finaljar" value="${project.build.directory}/payara-micro.jar" />
-                            <property name="install.dir.name" value="${install.dir.name}" />                            
-                        </ant>
-                    </tasks>
-                </configuration>
-            </execution>
-        </executions>
+            <groupId>org.jvnet.maven-antrun-extended-plugin</groupId>
+            <artifactId>maven-antrun-extended-plugin</artifactId>
+            <executions>
+                <execution>
+                    <phase>package</phase>
+                    <goals>
+                        <goal>run</goal>
+                    </goals>
+                    <configuration>
+                        <tasks>
+                            <ant dir="." antfile="build.xml" target="create.distribution">
+                                <property name="bundlename" value="fish.payara.micro" />
+                                <property name="finaljar" value="${project.build.directory}/payara-micro.jar" />
+                                <property name="install.dir.name" value="${install.dir.name}" />                            
+                            </ant>
+                        </tasks>
+                    </configuration>
+                </execution>
+            </executions>
         </plugin>
+        <plugin>
+            <artifactId>maven-jar-plugin</artifactId>
+            <executions>
+                <execution>
+                    <id>default-jar</id>
+                    <phase>never</phase>
```

Comment:
This was done to avoid maven creating an empty JAR in addition to the JAR built by ant. If someone knows of a better way please inform me!